VRDR/MortalityData.cs

```diff
@@ -250,7 +262,68 @@ public string RaceCodeToRaceName(string code)
                 t => LinqHelper.EqualsInsensitive(t.Key, key)
             ).Value;
         }
-
+        // JurisdictionCodes uses IJE-defined two-character string as key, and provides the code defined in the US Vital Records Jurisdictions (NCHS) Value Set 
+        // all codes are from FIPS-5-2 except for YC which is from U.S. Board on Geographic Names (USGS - GNIS).   This is handled in the one use of this Dictionary in the code below.
+         public Dictionary<string, string> JurisdictionCodes = new Dictionary<string, string>
```

Comment:
This comment needs to follow the proper C# style to be picked up by the xml doc generator; take a look at the linting output you see likely locally, or the GitHub auto warnings right below this (I see it saying: `Missing XML comment for publicly visible type or member 'MortalityData.JurisdictionCodes'`).